undetected_chromedriver/webelement.py

```diff
@@ -35,3 +60,25 @@ def __repr__(self):
         if strattrs:
             strattrs = " " + strattrs
         return f"{self.__class__.__name__} <{self.tag_name}{strattrs}>"
+
+
+def _recursive_children(element, tag: str = None, _results=None):
+    """
+    returns all children of <element> recursively
+
+    :param element: `WebElement` object.
+            find children below this <element>
+
+    :param tag: str = None.
+            if provided, return only <tag> elements. example: 'a', or 'img'
+    :param _results: do not use!
+    """
+    results = _results or set()
+    for element in element.children():
+        if tag:
```

Comment:
```py
if tag is None or element.tag_name == tag:
  results.add(element)
```
I think is less lines of code and improved readability

Reply:
if tag is None , as you suggest  and tag would be "" ,  than it would throw an Type Error.